### PR TITLE
ADD: Disable some Watch app elements when app is not reachable

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/w7vmv6bj4m40udgjjp23j991z8

[download apk](http://filestorage.bluewallet.io:1488/1597627842.apk)

**Comment by marcosrdz** [comment]
Simulator Screen Shot - Apple Watch Series 5 - 44mm - 2020-08-16 at 23 55 08

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/zzdrhghewjg6eqt24azfmqppag

[download apk](http://filestorage.bluewallet.io:1488/1597637183.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/v45cunw070ah07dxqq0whz4a2w

[download apk](http://filestorage.bluewallet.io:1488/1597640404.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/md3rm7h0g8a41mknncnv9dtvam

[download apk](http://filestorage.bluewallet.io:1488/1597669840.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/jvggggm1pktx3evgb34uw73me0

[download apk](http://filestorage.bluewallet.io:1488/1597680476.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/k529wj3ktycu9wu71ghg01wcjg

[download apk](http://filestorage.bluewallet.io:1488/1597715912.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/m9zxg45xc2u0r54y13p9tmymw0

[download apk](http://filestorage.bluewallet.io:1488/1597804572.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/p2ek7k2d5kh3k0n654cgf12078

[download apk](http://filestorage.bluewallet.io:1488/1597806081.apk)

File: WatchConnectivity.ios.js [hunk 1]
```diff
@@ -163,6 +159,7 @@ export default class WatchConnectivity {
             preferredBalanceUnit: wallet.getPreferredBalanceUnit(),
             receiveAddress: receiveAddress,
             transactions: watchTransactions,
+            xpub: wallet.getXpub() ? wallet.getXpub() : wallet.getSecret(),
```
Comment (Overtorment) [inline WatchConnectivity.ios.js:162]: are you sure that `getXpub` is always available here? you're calling it, maybe its better to check 
`wallet.getXpub ? wallet.getXpub() : ...` ?


`getXpub` is available only on descendants of `AbstractHDWallet` but not `AbstractWallet`

**Review by Overtorment — CHANGES_REQUESTED** [review]
ios build on appcenter failed, which is concerning. I rerunned it, lets see

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ngm48whaukdenzn48hd7pge9g4

[download apk](http://filestorage.bluewallet.io:1488/1597922723.apk)

**Pull request merged**